ChiModules/LinearBoltzmannSolver/lbs_compute_precursors.cc

```diff
@@ -0,0 +1,61 @@
+#include "lbs_linear_boltzmann_solver.h"
+
+#include "ChiMath/SpatialDiscretization/FiniteElement/PiecewiseLinear/pwl.h"
+
+#include "chi_log.h"
+extern ChiLog& chi_log;
+
+
+//###################################################################
+/**Compute the steady state delayed neutron precursor concentrations.*/
+void LinearBoltzmann::Solver::ComputePrecursors()
+{
+  auto pwl =
+      std::dynamic_pointer_cast<SpatialDiscretization_FE>(discretization);
+  const size_t J = max_precursors_per_material;
+
+  precursor_new_local.assign(precursor_new_local.size(), 0.0);
+
+  //================================================== Loop over cells
+  for (auto& cell : grid->local_cells)
```

Comment:
Might be silly, but can you make this `const auto&`, only if it does not conflict with something.

Reply:
Done.